prime-router/metadata/fhir_mapping/hl7/datatypes/XON/Organization.yml

```diff
@@ -11,17 +11,23 @@ name:
   expressionType: HL7Spec
 
 extension_1-nameType:
+  condition: xon2 NOT_NULL
   generateList: true
   expressionType: nested
+  vars:
+    xon2: XON.2
   expressionsMap:
     url:
       type: SYSTEM_URL
       value: organization-name-type
     valueCodeableConcept:
-      valueOf: datatype/CodeableConcept
+      valueOf: datatypes/CWE/Coding
```

Comment:
I'm a little confused as to how this works.... where does the `coding` property come from?

Reply:
`coding` comes from `CWE/CodeableConcept` which is referenced as an extension on `CWE/Coding`

Reply:
hmmm.. I see, should maybe the extension just get updated to be a `valueCoding`?

Reply:
yeah, good idea. Made the change